### ISSUE TITLE
Redesign or Rewrite profile page

### ISSUE THREAD

**Issue by roopeshsn** [roopeshsn/freshbey#94]
**Is your feature request related to a problem? Please describe.**
The profile page should be redesigned so that it will improve the UX.

**Describe the solution you'd like**
Here's the current profile page of Freshbey,

![profile_freshbey](https://user-images.githubusercontent.com/70762571/195482339-2e7c8fe5-d1ae-4cc9-ad25-03765b8c1111.png)

This page should be redesigned similarly to this.

![screenshot-www amazon in-2022 10 13-07_35_21](https://user-images.githubusercontent.com/70762571/195482385-a5b84066-8996-4192-a014-5dbed2e6892a.png)

As you see on Amazon's profile page things are arranged properly orders, addresses, etc.

**Describe alternatives you've considered**
_No response_

**Additional context**
_No response_

**Comment by ranjankuldeep** [comment]
Can u share some designs?
How screen will look after clicking Your orders and all?

**Comment by roopeshsn** [comment]
So we have user profiles and orders on the same page, right? When clicking "Your orders" a page should render with only orders. Use the same order table.

**Comment by roopeshsn** [comment]
Also, we only need "Your orders", "Login and security" and "Your addresses". Nothing else is required.

**Comment by ranjankuldeep** [comment]
ok Fine. I UNDERSTOOD
Please assign. @roopeshsn

**Comment by roopeshsn** [comment]
Are you working on this @ranjankuldeep?

**Comment by ranjankuldeep** [comment]
I already have an issue assigned.
I am waiting for the review of my PR #95 . and also to create a PR on issue #59.

After completing these two, I would be happy to work on this.

**Comment by roopeshsn** [comment]
This issue is still open!

**Comment by gkupdx** [comment]
@roopeshsn I would like to be assigned to this

**Comment by roopeshsn** [comment]
You can work on this @gkupdx!

**Comment by gkupdx** [comment]
@roopeshsn Am I supposed to create separate component screens for each path? For example, if a user clicks "My Orders", should it direct the user to a /profile/my-orders route? Or can the orders table be displayed on the profile page?

**Comment by roopeshsn** [comment]
> @roopeshsn Am I supposed to create separate component screens for each path? For example, if a user clicks "My Orders", should it direct the user to a /profile/my-orders route? Or can the orders table be displayed on the profile page?

Yeah, you're right!

**Comment by roopeshsn** [comment]
We need the UI to be clear and concise not cluttered with the whole data.

**Comment by gkupdx** [comment]
@roopeshsn I had a question about the "Your Addresses" route. Where should I get the address data from? The app doesn't ask you to enter your address when you register for a new account so the profile page doesn't have access to that information.

**Comment by roopeshsn** [comment]
Yeah, the address is not stored in the DB. Need to implement that feature #98. You can leave the address part.

**Comment by SpreeZ00** [comment]
Hi, I would like to try this. I'll set up the dev environment in the next days and could start working on this.

Did I get you right, that you want three Amazon like tiles for "Your Orders", "Security" and "Your Addresses", that link to another page e.g. "/profile/security" etc.?

**Comment by roopeshsn** [comment]
> Hi, I would like to try this. I'll set up the dev environment in the next days and could start working on this.
> 
> Did I get you right, that you want three Amazon like tiles for "Your Orders", "Security" and "Your Addresses", that link to another page e.g. "/profile/security" etc.?

Yep, you're right!

**Comment by SpreeZ00** [comment]
> > Hi, I would like to try this. I'll set up the dev environment in the next days and could start working on this.
> > Did I get you right, that you want three Amazon like tiles for "Your Orders", "Security" and "Your Addresses", that link to another page e.g. "/profile/security" etc.?
> 
> Yep, you're right!

Alright, then please assign me to this.
